package.json

```diff
@@ -41,12 +41,11 @@
     "express-winston": "^2.0.0",
     "helmet": "^3.1.0",
     "images-require-hook": "^1.0.3",
+    "immutable": "^3.8.1",
```

Comment:
Using Immutable for the config storage. There are a few reasons for this:

- It has `get` and `set` methods like nconf, so we won't need to change calls to`conf`
- It is universal-friendly
- I think what we need is not a config management tool, but a generic data structure tool
- We wanted an opportunity to try Immutable out

We could just code our own implementation, but it